ci_framework/roles/set_openstack_containers/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: 'Make sure {{ cifmw_set_openstack_containers_basedir }} exists'
+  ansible.builtin.file:
+    path: "{{ cifmw_set_openstack_containers_basedir }}/artifacts"
+    state: directory
+
+- name: Set container tag from dlrn md5 hash file
+  when:
+    - cifmw_set_openstack_containers_tag_from_md5 | bool
+  block:
+    - name: Get the md5 hash from hash file
+      ansible.builtin.slurp:
+        src: "{{ ansible_user_dir }}/ci-framework-data/artifacts/repositories/delorean.repo.md5"
```

Comment:
That should use `cifmw_set_openstack_containers_basedir` IMHO - and probably a check should be done in order to ensure it actually exists.
Note that ppl may override the location, and it may very well be located elsewhere... Using a parameter for the whole basedirectory tree would make sense.

Reply:
ack